<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battlewindow</class>
 <widget class="QWidget" name="battlewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="3" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_minion11">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion12">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion13">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion14">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion15">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion16">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion17">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="6" rowspan="4" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pushButton_card_set2">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>111</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="剩余卡牌：30"/>
       </property>
       <property name="text">
        <string>牌库</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_end">
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>回合结束</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pushButton_card_set1">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>111</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="剩余卡牌：30"/>
       </property>
       <property name="text">
        <string>牌库</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="5" rowspan="2" colspan="3">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>209</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>158</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_hands1">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands2">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands3">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands4">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands5">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands6">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands7">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands8">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands9">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hands10">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lanel_mana">
       <property name="text">
        <string>水晶：</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="3" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>108</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_minion21">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion22">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion23">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion24">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion25">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion26">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_minion27">
         <property name="minimumSize">
          <size>
           <width>61</width>
           <height>101</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>116</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7" rowspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>178</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="2" colspan="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>160</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" rowspan="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>318</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_weapon1_2">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hero2">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_skill2">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>278</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="3" rowspan="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>318</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_weapon1">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hero1">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_skill1">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>278</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" rowspan="8" colspan="3">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>61</width>
       <height>521</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_16">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_17">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_18">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_19">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_20">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_25">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_24">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_23">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_22">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_21">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>81</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
